frontend/tablet/src/components/Button.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
The margins are not the `Button`'s responsibility as a component. They can have an optional `style` prop (type `ViewStyle`) supplied to them instead. That is a better approach that allows for broader use. Replace the `margin` prop with `style`, but don't use it where you render the buttons just yet. I will create a component that will take care of this special case you will see how later and it will make sense I promise. :-D